packages/neuron-wallet/src/controllers/sudt.ts

```diff
@@ -34,4 +35,27 @@ export default class SUDTController {
       result: { tokenID: params.tokenID, symbol: symbol, tokenName: name, decimal: decimal },
     }
   }
+
+  public async getUDTTokenInfoAndBalance(params: {
+    tokenID: string
+    holder: string
+    udtType: UDTType
+  }): Promise<Controller.Response<SudtTokenInfo & { balance: string; capacity: string }>> {
+    const { tokenID, holder, udtType } = params
+    const info = await this.getSUDTTokenInfo({ tokenID })
+    if (!info.result) {
+      return {
+        status: ResponseCode.Fail,
+      }
+    }
+    const balance = await AssetAccountService.calculateUDTAccountBalance(holder, tokenID, udtType)
+    const ckbBalance = await AssetAccountService.calculateAvailableCKBBalance(holder)
+
+    const capacity = BigInt(MIN_SUDT_CAPACITY) + BigInt(ckbBalance)
```

Comment:
`MIN_SUDT_CAPACITY` is an assumption, the cell can receive CKB so its capacity could be more than `MIN_SUDT_CAPACITY`